### DISCUSSION TITLE
Emulate Zigbee End-Device

### DISCUSSION THREAD

**Discussion by tjunussov** [Koenkk/zigbee2mqtt#12982] [Devices]
Create virtual End-Device ( Light, Dimmer, Sensor etc ) and expose them to Zigbee Network, something like reversed to current approach.

The use case is: There are some nice looking Touch Panels ( like Aqara Smart Scene Panel Switch S1 or Orvibo MIXPAD and etc.. )
they act like coordinators, but they are compatible with only own branded devices. So maybe it is possible to emulate these devices and pass it MQTT network. 

It will open wide variety of cases ( Control your virtual/actual devices from brands native applications, Aqara, Samsung, Philips, Google, Yandex Station and etc...)  

Config Example: 

{
     "ORVIBO_SM10ZW_DOOR_SENSOR":{
        modelId: 75a4bfe8ef9c4350830a25d13e3ab068", 
        default_value: 0, 
}
  
p.s Maybe here we talking about something like two-way Zigbee Bridge ? Expose all entities of HA to Zigbee Network, I also asked question here [https://community.home-assistant.io/t/zha-as-a-zigbee-end-device/273957](https://community.home-assistant.io/t/zha-as-a-zigbee-end-device/273957)

**Comment by tjunussov** [comment]
MQTT2Zigbee

**Comment by kristian** [comment]
I would be very interested in this. Has there been any efforts / research in the open source community about this meanwhile? Thanks!
